lib/mutant/result.rb

```diff
@@ -269,6 +270,15 @@ def timeout?
         !isolation_result.timeout.nil?
       end
 
+      # Test for unexpected process abort
+      #
+      # @return [Boolean]
+      def process_abort?
+        process_status = isolation_result.process_status or return false
+
+        !timeout? && !process_status.exited?
```

Comment:
The reason to not check for abnormal process exits on timeouts, is  that on timeouts mutant *causes* an abnormal exit by itself via sending the process a `SIGKILL`.

Timeouts are a separate coverage criteria, hence "shadow" the result from being eligible from abnormal process exit coverage criteria.